src/services/item/plugins/importExport/service.ts

```diff
@@ -361,6 +374,33 @@ export class ImportExportService {
     }
   }
 
+  /**
+   * Extract the app settings from the export items and attach them to the existing items in the DB.
+   */
+  private async insertAppSettings(
+    dbConnection: DBConnection,
+    actor: MinimalMember,
+    uploadedItems: ItemRaw[],
+    items: GraaspExportItem[],
+  ) {
+    await Promise.all(
+      uploadedItems.map((uploadedItem, idx) => {
+        if (uploadedItem.type === ItemType.APP) {
+          const appSettings = items[idx].appSettings;
+          return Promise.all(
+            appSettings!.map((appSetting) => {
+              // Remove the id property from the imported app settings as a precaution. Remove this condition as soon as the id is stripped directly in the post function.
+              if (appSetting['id']) {
+                delete appSetting['id'];
+              }
+              return this.appSettingService.post(dbConnection, actor, uploadedItem.id, appSetting);
+            }),
```

Comment:
I guess it would make sense to have a post many? It could post all appSettings of all the items at once.